python/dgl/data/chem/csv_dataset.py

```diff
@@ -37,14 +38,16 @@ class CSVDataset(object):
     cache_file_path: str
     Path to store the preprocessed data
     """
-    
+
     def __init__(self, df, smile2graph=smile2graph, smile_column='smiles', cache_file_path="csvdata_dglgraph.pkl"):
         if 'rdkit' not in sys.modules:
             from ...base import dgl_warning
-            dgl_warning("Please install RDKit (Recommended Version is 2018.09.3)")
+            dgl_warning(
+                "Please install RDKit (Recommended Version is 2018.09.3)")
```

Comment:
I think you can also include the recommended installation way here, so that users do not need to spend additional time for searching.